ofl/abeezee/DESCRIPTION.en_us.html

```diff
@@ -3,4 +3,4 @@
 italic carefully reminds young readers of fluent writing movements
 and inspires them to create their own unique handwriting.</p>
 
-<p>Learn more at <a href="http://www.carrois.com">carrois.com</a>.</p>
\ No newline at end of file
+<p>Learn more at <a href="http://www.carrois.com">carrois.com</a>.</p>
```

Comment:
You can add the "Contribute blabla + url git repo" at the end + empty line at the end